src/bootstrap.js

```diff
@@ -185,7 +186,8 @@ module.exports = (ceConfig = {}, pnConfig = {}) => {
         }
 
         if (channel.indexOf(ChatEngine.ceConfig.globalChannel) === -1) {
-            channel = [ChatEngine.ceConfig.globalChannel, 'chat', chanPrivString, channel].join('#');
+            // TODO: DIS BISH GOTTA GO
```

Comment:
Add GH issues or Jira tickets instead of inline TODOs

Reply:
Removed